roles/jenkins/templates/seed_job.groovy

```diff
@@ -0,0 +1,61 @@
+import jenkins.model.*
+
+def jobName = "{{ jenkins_seed_job.name }}"
+
+def configXml = """\
+  <?xml version='1.0' encoding='UTF-8'?>
+  <project>
+    <actions/>
+    <description></description>
+    <keepDependencies>false</keepDependencies>
+    <properties>
+    </properties>
+    <scm class="hudson.plugins.git.GitSCM" plugin="git@3.6.4">
```

Comment:
Hardcoding the version in xml is uggly and will eventually become out of sync. Either get the installed git version from jenkins and inject it in the template, or create the seed job as pure groovy dsl.

Reply:
Variables are now implemented, about pure groovy or not we can refactor this later if we feel it makes sense